plugin.video.ad6/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.ad6" name="Alpe d&#39;HuZes" version="0.1.0" provider-name="Menco Bolt">

```

Comment:
can you avoid the html character reference and just use ascii ' ?

Reply:
Done. But yo kodi generated this. Also in my own setup it works ok.